scio-core/src/main/scala/com/spotify/scio/pubsub/PubSubAdmin.scala

```diff
@@ -19,7 +19,7 @@ package com.spotify.scio.pubsub
 
 import com.google.pubsub.v1.PublisherGrpc.PublisherBlockingStub
 import com.google.pubsub.v1.SubscriberGrpc.SubscriberBlockingStub
-import com.google.pubsub.v1.{GetTopicRequest, PublisherGrpc, SubscriberGrpc, Subscription, Topic}
+import com.google.pubsub.v1._
```

Comment:
Hi @BenFradet can you keep the expanded version here?

Reply:
sure :+1: